src/modules/access/components/wallets_lists/WalletsList.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
+    <p class="title5 mb-5">All Wallet Options</p>
+    <!-- Filter -->
+    <div class="flex gap-4 mb-5 flex-wrap">
+      <button
+        v-for="filter in filterOptions"
+        :key="filter.value"
+        @click="clickFilter(filter.value)"
+        class="bg-primary text-white text-center rounded-full py-1 px-4 min-w-[100px]"
+      >
+        {{ filter.name }}
+      </button>
+    </div>
+    <!-- Search -->
+    <div class="mb-5 max-w-[600px]">
+      <SearchInput @search="searchWallet" />
+    </div>
+    <!-- Wallets-->
+    <div
+      class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-5 gap-4"
+    >
+      <div
+        v-for="wallet in displayWallets"
+        :key="wallet.id"
+        class="flex flex-col items-center bg-white p-4 rounded-lg hoverOpacityHasBG cursor-pointer"
+        @click="clickWallet(wallet)"
+      >
+        <AsyncImg
+          :asyncImg="wallet.iconUrl"
+          :alt="wallet.name"
+          class="rounded-lg"
+        />
+        <p class="text-info pt-2">{{ wallet.name }}</p>
+      </div>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import * as rainndowWallets from '@rainbow-me/rainbowkit/wallets'
+import type { Wallet } from '@rainbow-me/rainbowkit'
+import AsyncImg from './AsyncImg.vue'
+import SearchInput from './SearchInput.vue'
+import { ROUTES_HOME } from '@/router/routeNames'
+import IconKeystore from '@/assets/icons/software_wallets/icon-keystore-file.svg'
+import IconMnemonic from '@/assets/icons/software_wallets/icon-mnemonic.svg'
+import IconPrivateKey from '@/assets/icons/software_wallets/icon-private-key-grey.png'
+import { useRouter } from 'vue-router'
+/** -------------------
+ *  Rainbow Wallets
+ * -------------------*/
+const WC_PROJECT_ID = import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID
+
+const DEFAULT_IDS = ['enkrypt', 'mew']
+const wallets: Wallet[] = []
+interface RainbowWalletOptions {
+  projectId: string
+}
+type CreateWalletFn = (config: { projectId: RainbowWalletOptions }) => Wallet
+const walletRecords = rainndowWallets as unknown as Record<
+  string,
+  CreateWalletFn
+>
+
+for (const key in walletRecords) {
+  if (Object.hasOwnProperty.call(walletRecords, key)) {
```

Comment:
technically should be `Object.prototype.hasOwnProperty`, but works either way

Reply:
Fixed